### PR TITLE
NO-ISSUE - Tip users to run `make lint` on linter errors

### PR DESCRIPTION
**Pull request by eliorerz**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -479,7 +479,7 @@ flake8:
 	skipper make _flake8
 
 _flake8:
-	flake8 $(FLAKE8_EXTRA_PARAMS) $(LINT_CODE_STYLING_DIRS)
+	flake8 $(FLAKE8_EXTRA_PARAMS) $(LINT_CODE_STYLING_DIRS) || (tput setaf 3; echo "If you keep seeing this error[s] try to make reformat"; tput sgr0; exit 1)
```
Comment (osherdp) [inline Makefile:482]: ``tput``? what are we animals? :joy:

Comment (eliorerz) [reply Makefile:482]: @osherdp Sorry, I just saw it. Any better suggestion instead of `tput`?

Comment (osherdp) [reply Makefile:482]: no it's ok
I'm just joking

**Pull request merged**